doc/source/devel/howto_recorders.rst

```diff
@@ -42,6 +42,10 @@ Sardana provides the following standard recorders (grouped by types):
     * JsonRecorder [*]
     * OutputRecorder
 
+* generic [*]
+.. TODO Need to document the implementation and configuration
```

Comment:
Need at least one blank line before the `.. TODO` comment otherwise travis complains